pet-kata/src/test/java/org/eclipse/collections/petkata/Exercise4Test.java

```diff
@@ -69,6 +69,8 @@ public void getAgeStatisticsOfPets()
         Assert.assertEquals(stats.getCount(), petAges.size());
 
         // Hint: JDK xyzMatch = Eclipse Collections xyzSatisfy
+        // Use IntPredicates, lambda or both?
+        // Is it possible to add a test in order to determine the average without using stats?
```

Comment:
I'm not sure what in intended by the last comment on average. Can you clarify please?

Reply:
The solution also contains the following assertion:
`Assert.assertEquals(2.0d, petAges.median(), 0.0);`
It is not clear from the exercise that this is requested as a possible answer.
To me, it looks like it is a similar assertion as the one above:
`Assert.assertEquals(stats.getAverage(), petAges.averageIfEmpty(0), 0.0);`

I wanted to clarify this, but since you have a question about it, it misses the point ;-) So, maybe we should leave it out or do you have a suggestion for clarifying this?

Reply:
Ah I see, thanks for clarifying. I did not realize median was included in the solutions. We should probably remove it from the solutions unless we add the code to implement median using the JDK in the kata. I would make this a separate test. We can do this work in a separate PR. I think this looks good to merge now, thank you!